.gitmodules

```diff
@@ -0,0 +1,7 @@
+[submodule "cpython-static"]
+	path = src/submodule_packages/cpython-static
+	url = git@github.com:guyush1/cpython-static.git
```

Comment:
You are probably missing the branch name, no?

Reply:
The actual commit/branch is set in the repository, not in the submodule file.

Reply:
Ah, i mist have been confused since the other submodule does contain a branch name. But i gyess it dows not matter